resources/views/dinnerform/admin_includes/dinnerform-details.blade.php

```diff
@@ -0,0 +1,93 @@
+<form method="post"
+      action="{{ ($dinnerformCurrent == null ? route("dinnerform::add") : route("dinnerform::edit", ['id' => $dinnerformCurrent->id])) }}"
```

Comment:
Maybe change this to isset()
```suggestion
      action="{{ (!isset($dinnerformCurrent) ? route("dinnerform::add") : route("dinnerform::edit", ['id' => $dinnerformCurrent->id])) }}"
```